components/Post.php

```diff
@@ -8,7 +8,7 @@
 class Post extends ComponentBase
 {
     /**
-     * @var RainLab\Blog\Models\Post The post model used for display.
+     * @var BlogPost The post model used for display.
```

Comment:
Why'd you change this?

Reply:
PhpStorm complained about this import and BlogPost is already imported.